cms/envs/common.py

```diff
@@ -485,7 +485,7 @@
     # .. toggle_default: False
     # .. toggle_description: Whether to replace ID verification course/certificate requirement
     # with an in-course Honor Code agreement
-    # (https://github.com/edx/edx-name-affirmation)
+    # (https://github.com/openedx/edx-name-affirmation)
```

Comment:
This is still wrong.
```suggestion
    # (https://github.com/edx/edx-name-affirmation)
```

Reply:
huh, I wonder why that's not going through, I definitely committed it via the UI. I will do it locally.